lionweb/src/test/kotlin/com/strumenta/kolasu/lionweb/LionWebModelConverterTest.kt

```diff
@@ -840,6 +827,44 @@ class LionWebModelConverterTest {
         assertEquals(2, ((deserializedNode3.destination as CompositeDestination).elements[1] as SimpleRoot).id)
         assertEquals(Position(Point(1, 2), Point(3, 4)), deserializedNode3.position)
     }
+
+    @Test
+    fun canSerializeAndDeserializePlaceholderNodes() {
+        val kl = KolasuLanguage("my.language").apply {
+            addClass(SimpleRoot::class)
+        }
+        val mc = LionWebModelConverter()
+        mc.exportLanguageToLionWeb(kl)
+
+        val node1 = SimpleRoot(1, mutableListOf()).setSourceForTree(LionWebSource("MySource1"))
+        val node2 = SimpleRoot(2, mutableListOf()).setSourceForTree(LionWebSource("MySource2"))
+        node1.origin = MissingASTTransformation(node2)
+        val lwNode1 = mc.exportModelToLionWeb(node1)
+        // We verify the exported data is correct
+        val lwNode1Origins = lwNode1.getReferenceValueByName("originalNode")
```

Comment:
I was not expecting that: I thought the reference was on the annotation, not on the note itself

Reply:
OK, sorry, both the annotation and the node have two references called "originalNode"

Reply:
Yes, but the annotation's one is not used. I used a trick because in general, while annotations can have references in Lionweb, Kolasu's reference resolver only works on Nodes. So while in Kolasu we have `node --> missing_origin --> origin`, in Lionweb I represented this as `node with annotation --> origin`.